src/interface_adapter/round/RoundState.java

```diff
@@ -87,4 +104,78 @@ public void setMaxRounds(int maxRounds) {
     public void setCurrentRoundNumber(int currentRoundNumber) {
         this.currentRoundNumber = currentRoundNumber;
     }
+
+    public String getMultipleChoice1() {
+        return multipleChoice1;
+    }
+
+    public String getMultipleChoice2() {
+        return multipleChoice2;
+    }
+
+    public String getMultipleChoice3() {
+        return multipleChoice3;
+    }
+
+    public String getMultipleChoice4() {
+        return multipleChoice4;
+    }
+
+    public ArrayList<String> getEasyChoices () {
+        ArrayList<String> ret = new ArrayList<>();
+        ret.add(multipleChoice1);
+        ret.add(multipleChoice2);
+        Collections.shuffle(ret);
+        return ret;
+    }
+
+    public ArrayList<String> getMediumChoices () {
+        ArrayList<String> ret = new ArrayList<>();
+        ret.add(multipleChoice1);
+        ret.add(multipleChoice2);
+        ret.add(multipleChoice3);
+        ret.add(multipleChoice4);
+        Collections.shuffle(ret);
+        return ret;
+    }
+
+    public void setMultipleChoice1(String multipleChoice1) {
+        this.multipleChoice1 = multipleChoice1;
+    }
+
+    public void setMultipleChoice2(String multipleChoice2) {
+        this.multipleChoice2 = multipleChoice2;
+    }
+
+    public void setMultipleChoice3(String multipleChoice3) {
+        this.multipleChoice3 = multipleChoice3;
+    }
+
+    public void setMultipleChoice4(String multipleChoice4) {
+        this.multipleChoice4 = multipleChoice4;
+    }
+
+    public boolean isEasyRound() {
+        return isEasyRound;
+    }
+
+    public boolean isMediumRound() {
+        return isMediumRound;
+    }
+
+    public boolean isHardRound() {
+        return isHardRound;
+    }
+
+    public void setEasyRound(boolean easyRound) {
+        isEasyRound = easyRound;
+    }
+
+    public void setMediumRound(boolean mediumRound) {
+        isMediumRound = mediumRound;
+    }
+
+    public void setHardRound(boolean hardRound) {
+        isHardRound = hardRound;
+    }
```

Comment:
```suggestion
    public boolean isMultipleChoiceRound() {
        return multipleChoiceOptions != null && !multipleChoiceOptions.isEmpty();
    }
```
In conjunction with the above, now there is only one "source of truth" for deciding whether we show a multiple choice round or not.